### ISSUE TITLE
BluemiraWire.center_of_mass returning FreeCAD Vector

### ISSUE THREAD

**Issue by hsaunders1904** [Fusion-Power-Plant-Framework/bluemira#769]
## Describe the bug

The `center_of_mass` property on `BluemiraWire` is returning a FreeCAD `Vector` where it should be returning a `np.ndarray`.

## Steps to reproduce

```python
import numpy as np
from bluemira.geometry.tools import make_circle

circle = make_circle()
assert type(circle.center_of_mass) == np.ndarray
```

We may want to do a quick check that we're not accidentally exposing any other instances of FreeCAD types in the CAD API.

**Comment by ivanmaione** [comment]
@hsaunders1904 I think in your code, the assert should be on `type(circle.center_of_mass)`. However, I am not sure that we agreed on a standard convention for vectors (sometimes they are implemented as `list`, as `tuple`, as `np.ndarray`). In any case, the same format should be used for both input and output (of course, the format in which they are transformed internally to a function, it is not relevant). @DanShort12 @CoronelBuendia, any thoughts?

**Comment by hsaunders1904** [comment]
Ah, yes, that should have been `assert type(circle.center_of_mass) == np.ndarray`. Thanks, I've updated it.

The existing type-hinting for the function says that it should be returning a numpy array, It was actually returning a `FreeCAD.Base.Vector`. That type-hint was why I asserted it should be a numpy array here.

https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/773fc2f021fe52adcfe997fabf8fb40819e0244c/bluemira/codes/_freecadapi.py#L468-L472
